lib/presentation/widgets/home_presentation/home_page_view_display.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+
+import '../../../core/utils/ui_constants/keys.dart';
+import '../../../core/utils/ui_constants/text_styles.dart';
+import '../../bloc/bloc_impl.dart';
+import '../../view/home.dart';
+import 'home_page_view.dart';
+import '../general_widgets/movie_container.dart';
+
+class HomePageViewDisplay extends StatelessWidget {
+  HomePageViewDisplay({
+    super.key,
+  });
+
+  static const double generalPadding = 15;
+  static const double basicContainerWidth = 400;
+  static const double largeContainerHeight = 600;
+  static const double lilContainerWidth = 243;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView(
+      physics: const BouncingScrollPhysics(),
+      children: [
+        Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.only(bottom: generalPadding),
+              child: Center(
+                child: Text(
+                  'Popular',
+                  style: MovieTextStyles.categoriesTextStyle,
+                ),
+              ),
+            ),
+            Container(
+              width: basicContainerWidth,
+              height: largeContainerHeight,
+              child: MoviePageView(
+                key: homePageViewPopularKey,
+                bloc: BlocImpl(),
```

Comment:
You are creating the same instance 4 times

Reply:
why the same instance? aren't those different instances?